ui/app/templates/components/search-select.hbs

```diff
@@ -0,0 +1,58 @@
+{{#if shouldUseFallback}}
+  {{component
+    fallbackComponent
+    label=label
+    onChange=(action "onChange")
+    inputValue=inputValue
+    warning=warning
+    helpText=helpText
+  }}
+{{else}}
+  <label for="{{name}}" class="title is-4" data-test-field-label>
+    {{label}}
+    {{#if helpText}}
+      {{#info-tooltip}}{{helpText}}{{/info-tooltip}}
+    {{/if}}
+  </label>
+  {{#power-select
+    options=options
+    onchange=(action "selectOption")
+    placeholderComponent="search-select-placeholder"
```

Comment:
Oh this is interesting - when does this render just when things are being fetched?